share/availability/test/testing.go

```diff
@@ -99,6 +96,13 @@ func (dn *DagNet) Node() *Node {
 	return nd
 }
 
+// Node create a plain network node that can serve and request data.
+func (dn *DagNet) Node() *Node {
```

Comment:
[non-blocking][suggestion]

To be explicit and avoid any potential confusion, I like prepending `Test` to methods like this to ensure we don't accidentally call it from non Test purposes.

Reply:
We cannot import these methods from other packages, since it is a _test.go and has its own package - right @Wondertan ?

Reply:
@distractedm1nd I believe this is only true for exported methods in `_test.go`, but not true for methods exported inside a `test` pkg. I think go ignores `testdata` 

Not 100% sure re `test` pkg.

Reply:
> since it is a _test.go and has its own package - right 

It's not a `_test.go` file, but a `test` pkg and go does not make any assumption about `test` pkgs.

Reply:
I prefixed the structs and constructors with Test